nss-day-cohort-44/rare-rest-api-rare-frosty-flakes
# Jaron post models and fixtures

# Description
Created models, fixtures, viewSet, and URL mapping for posts.

## Issue ticket: 12

## Type of change

# Testing Instructions
`git fetch --all`
`git checkout jaron-post-models-and-fixtures`
`python3 manage.py runserver`

DONT FORGET TO RUN 
`./seed.sh`

User test has been set up for user 1 (steve). Tester should see 2 posts on the All Posts link.

`git branch -d jaron-post-models-and-fixtures`

# Checklist:
- [x] I have commented my code with at least "AUTHOR: FIRST LAST NAME" at the top, and then in hard-to-understand areas
